library/src/main/java/io/constructor/data/remote/ApiPaths.kt

```diff
@@ -1,7 +1,7 @@
 package io.constructor.data.remote
 
 object ApiPaths {
-    const val URL_AUTOCOMPLETE = "autocomplete/{value}"
+    const val URL_AUTOCOMPLETE = "autocomplete/%s"
```

Comment:
Can we change to match the other const values with `{term}`?

Reply:
this one is following the `getSearchResults` and `getBrowseResults` pattern by using `%s`, should I change those as well? @rubikzube

Reply:
As long as we are consistent one way or the other